packages/cashier/src/config/on-ramp-providers.ts

```diff
@@ -1,6 +1,16 @@
 import { localize } from '@deriv/translations';
+import { TProviderDetails, TProviderDetailsWithoutFrom, TServerError, TClientStore } from 'Types';
 
-const createBanxaProvider = store => ({
+export type TProviderResponse = {
+    error: TServerError;
+    service_token: {
+        banxa: {
+            url: string;
+        };
+    };
+};
+
+const createBanxaProvider = (store: any): TProviderDetails => ({
```

Comment:
```suggestion
const createBanxaProvider = (store: OnRampStore): TProviderDetails => ({
```
@hamid-deriv I think we can do like this, What do you think? 🤔